example_app_generator/generate_app.rb

```diff
@@ -13,6 +13,8 @@
 function_script_file = File.join(rspec_rails_repo_path, 'script/functions.sh')
 
 in_root do
+  prepend_to_file "Rakefile", "require 'active_support/all'"
```

Comment:
It's totally because rails isn't requiring this properly itself in our sample app, nothing we can really do here :/